config/client/i18n-languages.js

```diff
@@ -28,6 +29,11 @@ export default [
   { code: 'de', label: 'Deutsch' }
```

Comment:
What's the purpose of this section if we anyway pull from ` config/shared/locales.json` ?

I.e. why two separate lists? :-)

Reply:
I think `json` is a nice format for shared config, since it is fully compatible with both `require` (Node) and `import` (React); and we're kind of sure it won't import anything from other files.

1. server doesn't care about `label`. It only needs the code for validation.
2. I didn't want to delete all the unused labels, and json doesn't support comments (not a good enough reason)
3. We need to specify the default language for client (e.g. `config/client/i18n-languages`). (not a good reason to keep the list, just to keep the file)


So the reasons are not very strong. Maybe it's not nice to maintain and we should keep a single shared list in `json` with codes and labels. WDYT @simison?

Reply:
It's a good idea to keep it in a single place. I'm gonna make a commit with that and we can revert if there is a reason to.

Reply:
Yeah, I think we could have shared JSON with labels; I agree it's not ideal to keep labels for the backend but it's so much nicer to have just once canonical place and one source of truth that it's okay methinks.

JSON is perfect for this use indeed 👍

Reply:
Re: JSON not supporting comments:

You could add `README.md` or `LOCALES.md` in the same folder with the json file if you'd like to document it.

Reply:
ed43d582b4c6f4f2c622558a2ab3232fc67aeb51